### DISCUSSION TITLE
Preview Image node seems to take a long time to render the images

### DISCUSSION THREAD

**Discussion by kivS** [comfyanonymous/ComfyUI#1428] [Q&A] [answered]
Hello, after any prompt completion I need to wait a good chunk of time for the image preview node to refresh and load the images,
for context it's running on a collab on GCP with 15GB GPU RAM and 12GB system RAM, everything else runs with no issues.

Any pointers on what I could do to speed this up?

cheers

**Comment by marclove** [comment]
I'm having the same problem. Using LCM and the loading of the preview image takes 2-3x as long as the generation itself.

**Comment by ltdrdata** [comment, marked as answer]
When using Colab, the issue arises when transmitting through a low-performance network via tunnels and such, resulting in a painfully slow degradation of speed, especially when viewing slightly high-resolution images. In the settings, there are options to preview images in low resolution through image format and quality settings to address this issue.

![image](https://github.com/comfyanonymous/ComfyUI/assets/128333288/cc123be4-7a47-4dc1-92bf-45808e6a048f)

**Comment by marclove** [comment]
Ohhh…that totally makes sense. That fixed it! Thanks for the reply, @ltdrdata!
